tinymce/mavo-tinymce.js

```diff
@@ -24,12 +24,14 @@ Mavo.Elements.register(".tinymce", {
 				return;
 			}
 
+			const toolbarOptions = this.element.getAttribute("data-tinymce-toolbar")
```

Comment:
I would suggest using `mv-` instead of `data-` in the attribute name to make it more consistent with other Mavo-related stuff.

What do you think of using `toolbar` instead of `toolbarOptions`? Will it collide with some other variables (constants)?

Reply:
Great suggestions, thanks!